### PR TITLE
Check that include belongs to the same fugr

### PR DESCRIPTION
**Pull request by Fmueller17**
This pull request solves the issue #5099.

### CODE REVIEW

File: src/objects/zcl_abapgit_object_fugr.clas.abap [hunk 1]
```diff
@@ -1202,4 +1212,38 @@ CLASS zcl_abapgit_object_fugr IMPLEMENTATION.
...
+  METHOD belongs_incl_to_other_fugr.
+    " make sure that the include belongs to the function group
+    " like in LSEAPFAP Form TADIR_MAINTENANCE
+    DATA ls_tadir TYPE tadir.
+    DATA lv_namespace TYPE rs38l-namespace.
+    DATA lv_area TYPE rs38l-area.
+    DATA lv_include TYPE rs38l-include.
+
+    rv_belongs_to_other_fugr = abap_false.
+    IF iv_include(1) = 'L' OR iv_include+1 CS '/L'.
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_fugr.clas.abap:1226]: Namespace split/check is not correct. Includes are /.../L...

**Review by mbtools — CHANGES_REQUESTED** [review]
Namespaces

File: src/objects/zcl_abapgit_object_fugr.clas.abap (hunk 1 above)
Comment (Fmueller17) [reply src/objects/zcl_abapgit_object_fugr.clas.abap:1226]: @mbtools Hi Marc, why do think it is not ok? I implemented like the SAP Standard. iv_include+1 CS '/L' means search  '/L' after the first '/' (iv_include+0).

grafik

Comment (Fmueller17) [reply src/objects/zcl_abapgit_object_fugr.clas.abap:1226]: It is not possible to have more than 2 / in a program name.
grafik

It is also not possible to have 2 / and the first one is not the first character.
grafik

Comment (mbtools) [reply src/objects/zcl_abapgit_object_fugr.clas.abap:1226]: that's not it. in your code, `lv_namespace` isn't set. (sorry, my marker wasn't precise. )

Comment (mbtools) [reply src/objects/zcl_abapgit_object_fugr.clas.abap:1226]: you are right (i read it as `=`)

**Review by mbtools — APPROVED** [review]
👍

**Pull request merged**